bpf/Makefile

```diff
@@ -312,9 +312,9 @@ tests/bpf_nat_tests.o: tests/bpf_nat_tests.ll
 	@$(ECHO_CC)
 	$(QUIET) ${LLC} ${LLC_FLAGS} -filetype=obj -o $@ $(patsubst %.o,%.ll,$@)
 
-.PHONY: go_prog_test
-go_prog_test: $(BPF_TEST)
-	$(GO) test -tags privileged_tests -exec sudo ./tests/prog_test
+.PHONY: go_prog_ct_test
+go_prog_ct_test: $(BPF_TEST)
```

Comment:
I missed to update `.github/workflows/lint-bpf-checks.yaml`. I will fix that with my next PR.